x/datapool/keeper/pool.go

```diff
@@ -492,3 +484,61 @@ func (k Keeper) GetDataValidationCertificate(ctx sdk.Context, poolID, round uint
 
 	return cert, nil
 }
+
+func (k Keeper) BuyDataPass(ctx sdk.Context, buyer sdk.AccAddress, poolID, round uint64, payment sdk.Coin) error {
```

Comment:
We must distribute a DataPass to a seller every time it is sold. So, when this PR is merged, I will add it to distribute to sellers whenever DataPass is sold.